Modern/utilities/src/main/java/org/bonej/utilities/RoiManagerUtil.java

```diff
@@ -104,15 +103,11 @@ public static boolean isActiveOnAllSlices(final int sliceNumber) {
 	// endregion
 
 	// region -- Helper methods --
-
-	private static int clamp(final int value, final int min, final int max) {
-		if (Integer.compare(value, min) < 0) {
-			return min;
-		}
-		if (Integer.compare(value, max) > 0) {
-			return max;
+	private static int clamp(final int value, final int max) {
+		if (value < 0) {
```

Comment:
Why get rid of `min`?
Should be 
```
if ( value < min ){
    return min;
}
```

Reply:
... I realise the rationale for the change is that only 0 is ever used as `min` for this class, but I prefer to leave things explicit rather than implied.

Reply:
Agreed, the method is clearer from the outside with both parameters, I'll change it back.

Reply:
> I'll change it back.

Thanks - but change it to the ultra-simple version (i.e. not `Integer.compare()`, which obscures the purpose because to understand what is going on you have to remember which way around it works)

Reply:
Nevermind, just noticed that the whole method was removed later in this PR as unused.